testgrid/metadata/job.go

```diff
@@ -87,11 +87,13 @@ func (m Metadata) String(name string) (*string, bool) {
 func (m Metadata) Meta(name string) (*Metadata, bool) {
```

Comment:
nit: I think the function comment is wrong? something like s/returns the name key/returns the value as a Metadata?

Reply:
I think this is correct (returns the name key (aka m[name] if that value is a child object)...

Reply:
Let me see if I can improve the docs in the next commit